motr/io_req.c

```diff
@@ -826,6 +826,27 @@ static int ioreq_iomaps_parity_groups_cal(struct m0_op_io *ioo)
 	return M0_RC(0);
 }
 
+static bool is_parity_verify_mode(struct m0_client *instance)
+{
+	return instance->m0c_config->mc_is_read_verify;
+}
+
+static void set_paritybuf_type(struct m0_op_io *ioo)
+{
```

Comment:
Are these changed related to this topic?

Reply:
Sorry for a little mess. You are right, this change is from https://github.com/Seagate/cortx-motr/pull/671 which should be merged before this patch.